.github/workflows/ci.yml

```diff
@@ -0,0 +1,32 @@
+name: Update emacs-overlay
+
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  refresh-overlay:
+    name: Refresh inputs
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2.3.4
+
+    - name: Install Nix
+      uses: cachix/install-nix-action@v13
+      with:
+        nix_path: nixpkgs=channel:nixos-unstable
+
+    - name: Update inputs
+      run: |
+        ./update
+
+    - name: Push commit with updated inputs
+      run: |
+        git config user.name github-actions
+        git config user.email github-actions@github.com
+        git commit -am "Updated repos/melpa"
+        git pull --rebase
```

Comment:
This does not require any secrets to be setup afterwards, right?

Reply:
Yup, no secrets required